examples/resources/dbt_cloud_repository/resource.tf

```diff
@@ -0,0 +1,29 @@
+# In order to find the github_installation_id, you can log in to dbt Cloud, replace <dbt_cloud_url> by your dbt Cloud URL and run the following commands in the Google Chrome console:
+
+# ```
+# dbt_cloud_api_result = await (fetch('https://<dbt_cloud_url>/api/v2/integrations/github/installations/').then(res => res.json()));
+# console.log("github_application_id: " + dbt_cloud_api_result.filter(res => res["access_tokens_url"].includes("github"))[0]["id"]);
+# ```
+
+# Alternatively, you can go to the page https://<dbt_cloud_url>/api/v2/integrations/github/installations/ and read the value of "id"
+
+resource "dbt_cloud_repository" "github_repo" {
+  project_id = 1234
+  remote_url = "git://github.com/<github_org>/<github_repo>.git"
+  github_installation_id = 9876
```

Comment:
just so I understand fully, this is created in dbt Cloud when adding a Github repo? Does it not come in the response for the POST request for repositories?

Reply:
When using the "GitHub" choice when liking a repo, instead of using the "Git Clone" approach, dbt Cloud connects to the repo via a GH App.
![image](https://user-images.githubusercontent.com/8754100/225333027-50ad0962-ed3a-40ba-90a4-e46e757f7c38.png)

When using the UI, `github_installation_id` is automatically added to the first `POST` request. It is also returned if we `GET` the repo.

Reply:
So we could, in theory, retrieve the ID from `https://<dbt_cloud_url>/api/v2/integrations/github/installations/` in the provider if we wanted to? Rather than the user having to go and get it after doing a manual connection? Or am I misunderstanding the flow and that this already expects dbt Cloud <-> Github to be linked in order for this ID to exist?